src/components/PrimaryNavigation/PrimaryNavigation.tsx

```diff
@@ -266,7 +266,7 @@ export const PrimaryNavigation = React.forwardRef<
               {...getTestProps(testIds.toggleButton)}
             >
               <span className={classes.footToggleContainer}>
-                <SideBar role="img" aria-hidden width={24} height={24} />
+                <Sidebar aria-hidden />
```

Comment:
`role="img"` isn't needed since we have `aria-hidden` here.  Sizing by default is also 24x24, so this was a bit redundant.